torch_geometric/distributed/local_graph_store.py

```diff
@@ -18,6 +18,13 @@ def __init__(self):
         self._edge_attr: Dict[Tuple, EdgeAttr] = {}
         self._edge_id: Dict[Tuple, Tensor] = {}
 
+        # for partition info related to graph
```

Comment:
Would recommend documenting `node_pb`, `edge_pb`, and `meta`. Meta is a bit concerning since it seems to be a Dict[Any, Any] with no additional information.

Reply:
done